public/css/GroupChat.css

```diff
@@ -353,4 +367,4 @@ input[type='text'] {
 
 .form-check-input {
     cursor: pointer;
-}
+}
```

Comment:
I think I accidentally removed this then put it back.  Not quite sure what happened there.

Reply:
ok